### DISCUSSION TITLE
How about a bash shell?

### DISCUSSION THREAD

**Discussion by christopinka** [ScoopInstaller/Scoop#4709] [Help and Questions] [answered]
What about Windows Subsystem for Linux? What is the crossover and comparisons? What happens if I enable WSL and install scoop from there. Use whatever WSL has including the terminal and then supplement with scoop packages? Do they mix and match well?

Thanks

**Comment by christopinka** [comment]
Ok. Requires admin for sure. Ok

**Comment by christopinka** [comment, marked as answer]
Oh shoot. Install-able. Of course. https://scoopsearch.github.io/#/apps?bash

**Comment by rashil2000** [comment]
Scoop cannot be installed on Linux. But Windows programs can be used from WSL. And Scoop is a Windows program.

If you don't want WSL, you can get Bash by either installing Git, MSYS2 or Cygwin.
